Fix py3 test_build_chart_data_for_field_with_language

```
======================================================================
FAIL: test_build_chart_data_for_field_with_language (onadata.libs.tests.utils.test_chart_tools.TestChartTools)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/onaio/onadata/onadata/libs/tests/utils/test_chart_tools.py", line 259, in test_build_chart_data_for_field_with_language
    self.assertEqual(data['field_label'], "Type of Eat")
AssertionError: 'Type of Eat Fr' != 'Type of Eat'
- Type of Eat Fr
?            ---
+ Type of Eat
```
More information: https://travis-ci.org/onaio/onadata/jobs/364008644

Part of #1295

--- a/onadata/libs/tests/utils/test_chart_tools.py
+++ b/onadata/libs/tests/utils/test_chart_tools.py
@@ -6,6 +6,7 @@ import os
 import unittest
 from decimal import Decimal
 
+from collections import OrderedDict
 from rest_framework.exceptions import ParseError
 
 from onadata.apps.logger.models import XForm
@@ -255,7 +256,9 @@ class TestChartTools(TestBase):
             "tests", "fixtures", "good_eats_multilang", "1.xml")
         self._make_submission(path)
         field = find_field_by_name(self.xform, 'food_type')
-        data = build_chart_data_for_field(self.xform, field, language_index=1)
+        language_index = list(OrderedDict(field.label)).index('English')
+        data = build_chart_data_for_field(self.xform, field,
+                                          language_index=language_index)
         self.assertEqual(data['field_label'], "Type of Eat")
 
     def test_build_chart_data_for_field_with_language_on_non_lang_field(self):

